### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by hoi235** [Extended-Thunder/send-later#519] [General]
Would you like to make sure that Mozilla Thunderbird's Send Later problem will be solved if you are in the url, please

**Comment by jph-sendlater** [comment]
I deleted / closed your other four threads because they're presumably all about the same thing. However, I can't decipher what you're trying to say. If you're using an auto translator please try again with a more descriptive prompt. For what it's worth, I've had good experiences with [google's translator](https://translate.google.com/).
